aquamarine/src/particle_executor.rs

```diff
@@ -59,21 +60,25 @@ impl<RT: AquaRuntime> ParticleExecutor for RT {
     type Future = Fut<Option<Self>>;
     type Particle = (Particle, CallResults);
 
+    #[instrument(level = tracing::Level::INFO, skip_all)]
     fn execute(
         mut self,
         p: Self::Particle,
         waker: Waker,
         current_peer_id: PeerId,
         key_pair: KeyPair,
     ) -> Self::Future {
+        let span = Span::current();
         let (particle, calls) = p;
         let particle_id = particle.id.clone();
         let data_len = particle.data.len();
-        let span = tracing::info_span!("Execute");
+        let blockng_span =
+            tracing::info_span!(parent: &span, "Particle executor: blocking AVM call");
+        let async_span = tracing::info_span!(parent: &span, "Particle executor: async task");
```

Comment:
WDYT about an approach to follow `Struct::function` in naming the spans?

For example:
```suggestion
        let blockng_span =
            tracing::info_span!(parent: &span, "ParticleExecutor::execute::spawn_blocking");
        let async_span = tracing::info_span!(parent: &span, "ParticleExecutor::execute::task");
```

Reply:
Done